tests/fuzz/fuzz_binary_decode.cc

```diff
@@ -21,7 +21,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 	size_t ptrSize = size;
 
 	// get some random type
-	uint8_t typeIndex = ptr[0];
+	uint16_t typeIndex = ptr[0];
```

Comment:
Is this python 3 related?

Reply:
No, but I found it because there were never more than 255 types before.
so not related to python3, that's why it is in a separate commit